services/api-server/src/simcore_service_api_server/core/settings.py

```diff
@@ -137,7 +137,9 @@ class ApplicationSettings(BasicSettings):
     )
     API_SERVER_LOG_CHECK_TIMEOUT_SECONDS: NonNegativeInt = 3 * 60
     API_SERVER_PROMETHEUS_INSTRUMENTATION_ENABLED: bool = True
```

Comment:
- why `API_SERVER_PROMETHEUS_INSTRUMENTATION_ENABLED` is the variable you defined to enable something you call `ApiServerHealthChecker`

Reply:
Very good point! Thanks for pointing this out. The health checker is quite close to the prometheus instrumentation because they both "peek" into the app's `LogDistributor` to understand what's going on. But of course they should not be mixed, as that only causes confusion. Hence, I have factored out the instrumentation from the health checker.